gke/README.md

```diff
@@ -32,19 +50,13 @@ Each node is a configured to e2-highmem-4 machine type that has 4 vCPU and 32G m
 
 ## DNS setup
 
-- Make sure the managed SSL certificate is "ACTIVE" by checking ["Load balancing" in GCP](https://pantheon.corp.google.com/net-services/loadbalancing/advanced/sslCertificates/list?project=PROJECT_ID&sslCertificateTablesize=50). This can take minutes up to hours.
+- Make sure the managed SSL certificate is "ACTIVE" by checking ["Load balancing" in GCP](https://pantheon.corp.google.com/net-services/loadbalancing/advanced/sslCertificates/list?project=PROJECT_ID&sslCertificateTablesize=50). This can take minutes up to hours. Note the SSL need to be used in an Ingress to be "ACTIVE".
```

Comment:
Could you describe more what this means here? I'm not sure what using the SSL in an Ingress means...

Reply:
Add some more text and a screenshot for this.